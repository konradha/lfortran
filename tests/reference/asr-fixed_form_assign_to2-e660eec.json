{
    "basename": "asr-fixed_form_assign_to2-e660eec",
    "cmd": "lfortran --indent --fixed-form --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form_assign_to2.f",
    "infile_hash": "ac85cc11719c45a4dc882205861d93a7375f2ce2a85289f800d2170e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-fixed_form_assign_to2-e660eec.stdout",
    "stdout_hash": "3c43a4e31275fbcb7f4fec07486eb4ee475b130a80d7c0b4fefddfea",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}