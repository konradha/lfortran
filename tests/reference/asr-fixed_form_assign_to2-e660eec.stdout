label is 100
(TranslationUnit 
    (SymbolTable
        1
        {
            main:
                (Program 
                    (SymbolTable
                        2
                        {
                            k:
                                (Variable 
                                    2 
                                    k 
                                    Local 
                                    () 
                                    () 
                                    Default 
                                    (Integer 8 []) 
                                    Source 
                                    Public 
                                    Optional 
                                    .false.
                                )
                            
                        }) 
                    main 
                    [] 
                    [(GoToTarget 
                        100
                    )
                    (If 
                        (IntegerCompare 
                            (IntegerConstant 0 (Integer 4 [])) 
                            Lt 
                            (IntegerConstant 1 (Integer 4 [])) 
                            (Logical 4 []) 
                            (LogicalConstant 
                                .true. 
                                (Logical 4 [])
                            )
                        ) 
                        [(= 
                            (Var 2 k) 
                            (IntegerConstant 100 (Integer 8 [])) 
                            ()
                        )] 
                        []
                    )
                    (Select 
                        (IntegerBinOp 
                            (Var 2 k) 
                            Add 
                            (Cast 
                                (IntegerConstant 1 (Integer 4 [])) 
                                IntegerToInteger 
                                (Integer 8 []) 
                                (IntegerConstant 1 (Integer 8 []))
                            ) 
                            (Integer 8 []) 
                            ()
                        ) 
                        [(CaseStmt 
                            [(IntegerConstant 1 (Integer 4 []))] 
                            [(GoTo 
                                100
                            )]
                        )
                        (CaseStmt 
                            [(IntegerConstant 2 (Integer 4 []))] 
                            [(GoTo 
                                101
                            )]
                        )] 
                        []
                    )
                    (GoToTarget 
                        101
                    )
                    (Print 
                        () 
                        [(StringConstant 
                            "yes" 
                            (Character 1 3 () [])
                        )] 
                        () 
                        ()
                    )]
                )
            
        }) 
    []
)
